dbt_subprojects/dex/models/trades/superseed/platforms/bulletx_v2_superseed_base_trades.sql

```diff
@@ -0,0 +1,77 @@
+{{ 
+    config(
+        schema = 'bulletx_superseed',
```

Comment:
```suggestion
        schema = 'bulletx_v2_superseed',
```

Reply:
versions should always be in schema names. alias names should remain the same in the pipeline.